.pipelines/java_packaging.yml

```diff
@@ -103,11 +101,39 @@ stages:
         SourceFolder: 'out/$(Agent.OS)/RelWithDebInfo/java/build/libs'
         TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
+    - script: |
+        cd $(Build.ArtifactStagingDirectory)
+        jar xvf onnxruntime-extensions-0.11.0.jar
+      displayName: Unpack JAR to view contents
+
+    - task: DownloadPipelineArtifact@0
```

Comment:
Is there any needs for a dependency check to ensure the Windows artifacts is ready?

Reply:
Yeah, since we combine them in the MacOS job, it depends on the completion of Windows and Linux jobs to ensure their artifacts are ready.